src/components/Breadcrumbs/Breadcrumbs.stories.tsx

```diff
@@ -133,5 +133,6 @@ export const LongTextMenu: StoryObj<Args> = {
     const canvas = within(canvasElement);
     const dropdownMenuTrigger = await canvas.findByRole('button');
     userEvent.click(dropdownMenuTrigger);
+    // TODO: capture open menu in Chromatic
```

Comment:
For some reason this change broke the capturing of the open Menu state in Chromatic, tried the following to avail:
- `await new Promise((resolve) => setTimeout(resolve, 300));` in the play function to wait for loading
- `delay: 500` in the story chromatic properties to wait before snapping

Reply:
Got it snapped with the decorator padding